.github/workflows/firesim-run-tests.yml

```diff
@@ -82,17 +82,20 @@ jobs:
         run: ./.github/scripts/launch-manager-instance.py
       - name: Setup N Github Actions Runners on AWS instance
         run: ./.github/scripts/setup-manager-self-hosted.py
-
-  setup-manager:
-    name: setup-manager
-    needs: [setup-self-hosted-manager]
-    runs-on: ${{ github.run_id }}
-    steps:
-      - uses: actions/checkout@v2
       - name: Setup FireSim repo (.pem, build-setup.sh, AWS credentials, submodules) and CI daemons
         uses: ./.github/actions/initialize-manager
         with:
           max-runtime-hours: 10
+      - name: Catch potentially orphaned managers
+        if: ${{ failure() || cancelled() }}
+        run: .github/scripts/change-workflow-instance-states.py ${{ github.run_id }} terminate ${{ github.token }}
```

Comment:
In the past, I had to mess around with the `PERSONAL_ACCESS_TOKEN` vs the `GITHUB_TOKEN` (I don't know if the normal provided `GITHUB_TOKEN` had enough permissions). Just an FYI. This should get tested before merging to verify this works.

Reply:
This appears to work fine. The GHA runner setup not so much.